docs/README.md

```diff
@@ -469,23 +462,12 @@ kubectl get events --sort-by=.metadata.creationTimestamp -n openfaas-fn
 ```
 
 ##### Troubleshoot Network Policies
-The NetworkPolicy configuration is designed to work with a Kubernetes IngressController. If you are using a NodePort or LoadBalancer you have to deploy NetworkPolicy below.
-```yaml
-apiVersion: networking.k8s.io/v1
-kind: NetworkPolicy
-metadata:
-  name: gateway
-  namespace: openfaas
-spec:
-  policyTypes:
-    - Ingress
-  podSelector:
-    matchLabels:
-      app: gateway
-  ingress:
-    - from: []
-      ports:
-        - port: 8080
```

Comment:
@bartsmykla this policy would allow an ingress controller to reach the gateway on 8080, without it I don't see how the nginx can forward traffic to the gateway.

Reply:
I think I just annotated `kube-system` with `openfaas-system` label

Reply:
But it was a month ago, so I'm sure in 80% only.

Reply:
@alexellis do you see any issues if we allow access to the 8080 port from any namespace? I'm guessing the basic auth is turned on so it shouldn't be a problem. I don't like the idea of labeling `kube-system`.

Reply:
I think we should ask @alexellis - I don't have any problem with it.

Reply:
You could DoS the core services from a function if we allowed access.

If we didn't apply the label, what would this cover?

Reply:
We could allow traffic from nginx using a pod selector:

```yaml
apiVersion: networking.k8s.io/v1
kind: NetworkPolicy
metadata:
  name: openfaas
  namespace: openfaas
spec:
  policyTypes:
    - Ingress
  podSelector: {}
  ingress:
  - from:
    - namespaceSelector:
        matchLabels:
          role: openfaas-system
    - namespaceSelector:
        matchLabels:
          role: openfaas-fn
      podSelector:
        matchLabels:
          role: openfaas-system
    - podSelector:
        matchLabels:
          app.kubernetes.io/name: ingress-nginx
```